web/modules/custom/ys_base/js/ys_base.js

```
@@ -0,0 +1,76 @@
+        if (block.classList.contains('ys-base-counter-processed')) {
+          return;
+        }
+
+        block.classList.add('ys-base-counter-processed');
+
+        const valueElement = block.querySelector('[data-counter-value]');
+        const buttons = block.querySelectorAll('[data-counter-action]');
+
+        // Load saved value from localStorage.
+        const storageKey = 'ys_counter_value';
+        let currentValue = parseInt(localStorage.getItem(storageKey)) || 0;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)